### PR TITLE
remove GetPodStatsCommand

### PR DESCRIPTION
**Pull request by gao-feng**
Signed-off-by: Gao feng <omarapazanadi@gmail.com>

### CODE REVIEW

File: hypervisor/report.go [hunk 1]
```diff
@@ -70,22 +70,3 @@ func (ctx *VmContext) reportVmFault(cause string) {
 		Cause: cause,
 	}
 }
-
-func (ctx *VmContext) reportPodStats(ev VmEvent) {
-	response := types.VmResponse{
-		VmId:  ctx.Id,
-		Code:  types.E_POD_STATS,
```
Comment (laijs) [inline hypervisor/report.go]: E_POD_STATS should be also removed

File: hypervisor/vm.go [hunk 2]
```diff
@@ -678,40 +678,18 @@ func (vm *Vm) Tty(containerId, execId string, row, column int) error {
 	return nil
 }
 
-func (vm *Vm) Stats() *types.VmResponse {
-	var response *types.VmResponse
-
+func (vm *Vm) Stats() *types.PodStats {
 	//if nil == vm.Pod || vm.Pod.Status != types.S_POD_RUNNING {
 	//	return errorResponse("The pod is not running, can not get stats for it")
 	//}
```
Comment (laijs) [inline hypervisor/vm.go]: could you please also remove the these comments

**Comment by laijs** [comment]
good!

**Comment by gao-feng** [comment]
updated

**Pull request merged**